examples/extract_films.py

```diff
@@ -0,0 +1,42 @@
+import csv
+import random
+import time
+import numpy as np
+
+
+def random_vector(dim):
+    return [np.float32(random.random()) for _ in range(dim)]
+
+
+def main():
+    out = list()
+
+    with open("/home/yhz/Downloads/ml-latest-small/movies.csv", 'r') as f:
```

Comment:
If `movies.csv` is unavailable, this script is also not useful.  And the file path here...
By the way, I edit thousands of lines for title ends with `, The` of the result from `films.csv`